idle_scan.cc

```diff
@@ -547,7 +547,7 @@ static void initialize_idleproxy(struct idle_proxy_info *proxy, char *proxyName,
   int newipid;
   unsigned int i;
   char filter[512]; /* Libpcap filter string */
-  char name[MAXHOSTNAMELEN + 1];
+  char name[FQDN_LEN];
```

Comment:
Here again, just as elsewhere, we need to allocate 1 extra byte for the null terminator.